Spigot-Server-Patches/0247-Allow-simple-built-in-biome-swapping.patch

```diff
@@ -0,0 +1,202 @@
+From 043c82fce1bc3c1ffb54bb7111476de1b14c0f47 Mon Sep 17 00:00:00 2001
+From: June <junehanabi@gmail.com>
+Date: Wed, 18 Oct 2017 15:47:29 -0500
+Subject: [PATCH] Allow simple built-in biome swapping
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperConfig.java b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+index f5cb9799..a8bfe03d 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+@@ -23,6 +23,7 @@ import org.bukkit.configuration.InvalidConfigurationException;
+ import org.bukkit.configuration.file.YamlConfiguration;
+ import co.aikar.timings.Timings;
+ import co.aikar.timings.TimingsManager;
++import java.util.Arrays;
+ 
+ public class PaperConfig {
+ 
+@@ -276,4 +277,22 @@ public class PaperConfig {
+     private static void authenticationServersDownKickMessage() {
+         authenticationServersDownKickMessage = Strings.emptyToNull(getString("messages.kick.authentication-servers-down", authenticationServersDownKickMessage));
+     }
++    
++    // Allow built-in biome replacement
++    public static Map<String,String> biomeReplacement = new HashMap<String, String>();
++    private static void biomeReplacement() {
++        List<String> biomes = getList("biomeReplacement", Arrays.asList(new String[]{}));
++        
++        // Stop here if list is empty
++        if(biomes.isEmpty())
++            return;
++        
++        for(String biome : biomes) {
++            String[] keyValue = biome.split(":");
++            if(keyValue.length != 2)
++                continue; // Ignore, should be exactly 2 elements
```

Comment:
should probably print something out instead of ignoring completely, otherwise issues with the config file will be overlooked